### PR TITLE
Add support for `offset` parameter

### PR DESCRIPTION
**Pull request by mocca102**

### CODE REVIEW

File: src/modules/browse.js [hunk 1]
```diff
@@ -42,6 +43,11 @@ function createQueryParams(parameters, userParameters, options) {
       queryParams.page = page;
     }
 
+    // Pull offset from parameters
+    if (!helpers.isNil(offset)) {
```
Comment (sblaurock) [inline src/modules/browse.js:47]: What's the thinking on `isNil` vs `isInteger`?

File: src/modules/browse.js [hunk 2]
```diff
@@ -227,6 +233,7 @@ class Browse {
    * @param {string} filterValue - Filter value to display results from
    * @param {object} [parameters] - Additional parameters to refine result set
    * @param {number} [parameters.page] - The page number of the results
+   * @param {number} [parameters.offset] - The number of results to skip from the beginning
```
Comment (sblaurock) [inline src/modules/browse.js:236]: Let's be sure to document the clarification:
```
The number of results to skip from the beginning. Can't be used together with 'page'
```

File: src/modules/browse.js [hunk 3]
```diff
@@ -227,6 +233,7 @@ class Browse {
    * @param {string} filterValue - Filter value to display results from
    * @param {object} [parameters] - Additional parameters to refine result set
    * @param {number} [parameters.page] - The page number of the results
```
Comment (sblaurock) [inline src/modules/browse.js:235]: Let's be sure to document the clarification:

```
The page of results to return. Defaults to 1. Can't be used together with 'offset'`
```

**Review by sblaurock — COMMENTED** [review]
A few comments - looking good!

File: src/modules/browse.js (hunk 1 above)
Comment (mocca102) [reply src/modules/browse.js:47]: At first I just saw isNil used in the other cases in the code so I just used it as well 😅 But after looking we are sending all the params to the qs.stringify function anyway so a value like '1' which is not an integer should also be accepted. Am I right?

Comment (sblaurock) [reply src/modules/browse.js:48]: Good point about the rest of the checks using `isNil` - thanks for the clarification. I think we're good to go!

**Review by sblaurock — APPROVED** [review]

**Pull request merged**